app/src/main/res/values/strings.xml

```diff
@@ -572,4 +572,17 @@
   <string name="administrator_controls_fragment_test_activity_label">Administrator Controls Fragment Test Activity</string>
   <string name="select_a_topic_to_start">Select a Topic to Start</string>
   <string name="revision_navigation_cards_header">Continue Studying</string>
+  <string name="info_tab_content_description">Info tab</string>
+  <string name="revision_tab_content_description">After completing a lesson, revise your concepts in the revision tab.</string>
+  <string name="practice_tab_content_description">Practice tab</string>
+  <string name="lessons_tab_content_description">Start learning here in the lessons tab.</string>
+  <string name="topic_lessons_tab_spotlight_hint">Find all your lessons here</string>
+  <string name="topic_revision_tab_spotlight_hint">Revise your lessons here</string>
+  <string name="first_chapter_spotlight_hint">Tap to start a chapter</string>
+  <string name="exploration_exit_button_spotlight_hint">Exit anytime using this button. We will save your progress.</string>
+  <string name="voiceover_icon_spotlight_hint">Would you like %s to read for you? Tap on this button to try!</string>
+  <string name="voiceover_language_icon_spotlight_hint">Tap to change voiceover language</string>
+  <string name="onboarding_next_button_spotlight_hint">Next</string>
+  <string name="promoted_story_spotlight_hint">From now, you can see stories recommended for you here</string>
```

Comment:
```suggestion
  <string name="promoted_story_spotlight_hint">From now, you can see lessons recommended for you here</string>
```

Trying to keep the vernacular consistent here since terms like "stories" and "chapters" may confuse the reader when used interchangeably with the concept that would think of as a lesson.

Reply:
done